SwiftMessages/StoryboardSegue.swift

```diff
@@ -0,0 +1,316 @@
+//
+//  StoryboardSegue.swift
+//  SwiftMessages
+//
+//  Created by Timothy Moose on 5/30/18.
+//  Copyright © 2018 SwiftKick Mobile. All rights reserved.
+//
+
+import UIKit
+
+/**
+ A configurable `UIStoryboardSegue subclass that utilizes SwiftMessages to
```

Comment:
Missing a backtick after `UIStoryboardSegue`?

Reply:
Got it